.github/workflows/integration.yml

```diff
@@ -30,6 +30,9 @@ jobs:
           sudo sysctl -w vm.max_map_count=262144
 
       - name: Runs OpenSearch cluster
+        env:
```

Comment:
nice!
btw, you can also declare OPENSEARCH_VERSION as a global-level variable to avoid repeated declaration; something like [this](https://github.com/opensearch-project/security/blob/c06365ca94c9a1d15e85b578a1ae48168bf0bca7/.github/workflows/ci.yml#L11).

Reply:
Nice! I didn't know you could set it globally. Will use that!